### ISSUE TITLE
[result] 

### ISSUE THREAD

**Issue by LW091** [zcablii/LSKNet#61]
### What's the feature?

Why is it that I use the same configuration as you (also 8GPU) and the training result is only 80?
Will it be worse to use this result to run the test set?
Is there anything I still need to set up?

**Comment by zcablii** [comment]
What do u mean by "training result"? Did you train/test on multiscale splits?

**Comment by LW091** [comment]
> What do u mean by "training result"? Did you train/test on multiscale splits?

Sorry, maybe I didn't make it clear. I mean my validation set results are only 80.

**Comment by zcablii** [comment]
Did you train/test on multiscale splits?

**Comment by LW091** [comment]
Yes, I'm using ms_trainval.

**Comment by zcablii** [comment]
Did you train on trainval set and validate on val?

**Comment by zcablii** [comment]
So you train on trainval and validate on trainval as well?

**Comment by LW091** [comment]
> So you train on trainval and validate on trainval as well?

I see that the configuration you gave did not set the val set, so I only gave the trainval set, but my training process is similar to the log you gave (as shown below, but this is a screenshot of the log you gave)
So is it because I'm not using val set?
![图片1](https://github.com/zcablii/LSKNet/assets/76935611/633223e7-54dc-436a-a927-3b480d4ccc3d)

**Comment by zcablii** [comment]
I am not sure, you can try to submit your test result and see.

**Comment by LW091** [comment]
> I am not sure, you can try to submit your test result and see.

Okay, thanks for the answer
